pubpub/pubpub
# fix(#2902): rollback blueprintjs version to 3.26 to fix inability to edit media width through numeric input

## Issue(s) Resolved
#2902 

## Test Plan
1. Go to draft
2. Click on image
3. Try to manipulate the width using the numeric input (not the slider)
4. Should work



## Screenshots (if applicable)

## Optional

### Notes/Context/Gotchas
I stared at this for longer than I wanted to, could not figure out why it wasn't working. Just rewinded to an older version of blueprint (newer version happened because of my regenerating the lockfile in #2843), which seemed to restore the expected behavior.
Might be a regression in blueprint or something they changed, idk.

### Supporting Docs
